Source/WarpX.cpp

```diff
@@ -20,10 +20,10 @@
 #ifdef WARPX_USE_PSATD
 #   include "FieldSolver/SpectralSolver/SpectralSolver.H"
```

Comment:
And the same here, only one should be imported. `SpectralKSpace.H` should be imported with and without RZ.